templates/democracy/arapp.json

```diff
@@ -2,34 +2,34 @@
   "environments": {
     "default": {
       "registry": "0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1",
-      "appName": "democracy-kit.aragonpm.eth",
+      "appName": "democracy-template.aragonpm.eth",
       "network": "rpc"
     },
     "rinkeby": {
       "registry": "0x98Df287B6C145399Aaa709692c8D308357bC085D",
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
-      "appName": "democracy-kit.aragonpm.eth",
+      "appName": "democracy-template.aragonpm.eth",
```

Comment:
We'll have to re-deploy these ones, right?

Reply:
Yes, absolutely. Everything will need to be re-audited and re-deployed.

Reply:
yup